content/en/docs/multicluster-management/enable-multicluster/agent-connection.md

```diff
@@ -47,7 +47,7 @@ multicluster:
 
 {{< tab "KubeSphere has not been installed" >}}
 
-There is no big difference than installing a standalone KubeSphere if you define a host cluster before installation. Please note that the `clusterRole` in `config-sample.yaml` or `cluster-configuration.yaml` has to be set as follows:
+There is no big difference than installing a standalone KubeSphere if you define a host cluster before installation. Please note that the `clusterRole` in `config-sample.yaml` has to be set as follows:
```

Comment:
I have a question here. Why `cluster-configuration.yaml` is deleted here? My understanding is that If users already have a Kubernetes cluster, they can set `clusterRole` to `host` in `cluster-configuration.yaml` before installation. After they apply the file to install KubeSphere, the cluster will be the host cluster.